webpack/assets/javascripts/react_app/redux/API/APIRequest.js

```diff
@@ -13,14 +23,16 @@ export const get = async (
     payload: modifiedPayload,
   });
   try {
-    const { data } = await API.get(url, headers, params);
+    const response = await API.get(url, headers, params);
+    handleSuccess(response);
     dispatch({
       type: SUCCESS,
```

Comment:
Seems to me like the callbacks should get called after the actions (in other words, switch the order of the `handleSuccess` and `dispatch`.)  I wouldn't want a side effect to happen before I'm even notified of what caused the side effect.